### DISCUSSION TITLE
How to get meAsUser

### DISCUSSION THREAD

**Discussion by OualiS** [plhery/node-twitter-api-v2#217] [Q&A] [answered]
<img width="435" alt="image" src="https://user-images.githubusercontent.com/32158518/156884311-090a206c-c680-48fc-8079-5565f0b2f8d9.png">

I'm using the example for stream tweets but you're using a variable that we can't get ?

Can you tell me how to get meAsUser please

**Comment by alkihis** [comment, marked as answer]
Hi!

This variable can be obtained through a client with has an user context (with OAuth 1.0a login/OAuth 2.0 User context).

```ts
// OAuth 1.0a
// You can get all those keys in the twitter dev portal, it will be linked to your Twitter account
const userClient = new TwitterApi({ appKey: '', appSecret: '', accessToken: '', accessSecret: '' })
const meAsUser = await userClient.currentUser() 
```
